src/js/util/settings.js

```diff
@@ -60,3 +60,11 @@ exports.getSetting = function(userBlob, settingName, def) {
   }
   return def;
 };
+
+exports.getClearServers = function(servers) {
+  return _.map(servers, function(server) {
+    var o = _.pick(server, 'host', 'port', 'secure');
+    o.port = +o.port;

```

Comment:
Maybe add a comment about this, or use the explicit variant, which might be easier to read.